src/main/java/me/ryanhamshire/GPFlags/flags/FlagDef_AllowVillagerTrade.java

```diff
@@ -0,0 +1,62 @@
+package me.ryanhamshire.GPFlags.flags;
+
+import me.ryanhamshire.GPFlags.Flag;
+import me.ryanhamshire.GPFlags.FlagManager;
+import me.ryanhamshire.GPFlags.GPFlags;
+import me.ryanhamshire.GPFlags.MessageSpecifier;
+import me.ryanhamshire.GPFlags.Messages;
+import me.ryanhamshire.GriefPrevention.events.ClaimPermissionCheckEvent;
+import org.bukkit.entity.Player;
+import org.bukkit.entity.Villager;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.player.PlayerInteractEntityEvent;
+
+public class FlagDef_AllowVillagerTrade extends FlagDefinition {
```

Comment:
I think AllowVillagerTrading might be a better name

Reply:
Do you think we want a ``NoVillagerTrading`` flag for the servers who have the ``VillagerTradingRequiresPermission`` set to false?